databricks/sdk/config.py

```diff
@@ -473,3 +498,36 @@ def deep_copy(self):
         """Creates a deep copy of the config object.
         """
         return copy.deepcopy(self)
+
+    def get_compute(self):
+        return Compute(self)
+
+
+class Compute:
+    serverless: bool = False
+    cluster_id: Optional[str] = None
+
+    def __init__(self, cfg: Config):
+        if cfg.compute is not None and cfg.cluster_id is not None:
+            raise ValueError("Cannot specify both compute and cluster id in the config file")
+        elif cfg.compute is not None:
+            if cfg.compute == "serverless":
+                self.serverless = True
+            elif cfg.compute.startswith("cluster:"):
+                self.cluster_id = cfg.compute.split("cluster:")[1]
+            else:
+                raise ValueError("compute must be either 'serverless' or 'cluster:<cluster_id>'")
+        elif cfg.cluster_id is not None:
+            self.cluster_id = cfg.cluster_id
+
+    def is_empty(self):
+        return self.serverless is False and self.cluster_id is None
```

Comment:
i thought this would be convenient. The downstream code will need to either check
```
if config.compute is None and config.cluster_id is None:
```
or
```
compute = config.get_compute()
if compute.serverless is False and compute.cluster_id is None:
```
which is longer than using a single method